.gitignore

```diff
@@ -1,6 +1,5 @@
 node_modules
 npm-debug.log
-build
```

Comment:
Right for the .gitignore...  So do you want to `truffle compile` before using `readContracts` from the core ? Seems weird